blockchain/blockchain.go

```diff
@@ -1112,6 +1119,58 @@ func (bc *blockchain) runActions(acts block.RunnableActions, ws factory.WorkingS
 	return ws.RunActions(ctx, acts.BlockHeight(), acts.Actions())
 }
 
+func (bc *blockchain) pickAndRunActions(ctx context.Context, coinBaseSelp action.SealedEnvelope,
```

Comment:
Be aware of this coinbase removal PR: https://github.com/iotexproject/iotex-core/pull/501